portfolio/src/main/webapp/index.html

```diff
@@ -49,53 +50,133 @@ <h4>Software Engineering Intern</h4>
-      <section>
-          <div class="info">
-            <div class="title"><h2>About Me</h2></div>
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      <section id="home" class="landing">
+        <div class="content"> 
+          <h2>Hi there.<br> I'm <span> Kayla Leung </span></h2>
+          <button id="testbutton" class="button" onclick="document.location = '#aboutme'">About Me</button>
+        </div>
+      </section>
+    </div>
+
+    <div id="aboutme" class="menu">
+      <section class="banner">
+        <div class="info">
+             <div class="title">
+               <h1>About Me</h1>
+             </div>
+             <p>
+               <div>
+                 <img src="images/dolby.jpg" width="200" height="250" alt="Photo of Kayla in Museum">
+                 <img src="images/ada.png" width="200" height="250" alt="Photo of Kayla in Museum">
+               </div>
+               Hi there! I’m Kayla. I’m a 20 year old rising Junior at Carnegie Mellon University originally from East Brunswick, New Jersey. I'm majoring in Computer Science with an additional major in Psychology and a minor in Human-Computer Interaction.  I’m an aspiring software engineer interested in working on human centered technology at the intersection of Computer Science and Psychology. <br>nIn my free time, I enjoy photography, writing on my blog with updates about my life as a student, and reading various psychology related books!
               </p>
         </div> 
       </section>
     </div>
 
     <div id="education" class="menu">
-      <section>
-          <div class="info">
-            <div class="title"><h2>Education</h2></div>
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-              </p>
+      <section class="banner">
+        <div class="info">
+             <div class="title">
+               <h1>Education</h1>
+             </div>
+             <h3>
+               Carnegie Mellon University 
+               <span>Expected Graduation: May 2020 </span>
+            </h3>
+            <h4>
+               <span>Pittsburgh, PA</span> 
+               B.S. Computer Science, Psychology
+               </span> 
```

Comment:
I think you can delete this `</span>` since it doesn't have a matching start tag.